components/system/components/AvatarGroup.js

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+const STYLES_UL = css`
+  list-style-type: none;
+
+  :hover li:nth-child(n + 2) {
```

Comment:
What do you need this for?

Reply:
do you mean the `list-style-type`?

Reply:
No I mean this :hover li:nth-child(n + 2) {}

Why isn't the hover rule inside of the `li` element?

Reply:
the transition animation doesn't work when I move it to the `li` element

Reply:
Gotcha, we can refine this as we use it. Thanks ^.^

Reply:
sounds good!